docs/public/index.css

```diff
@@ -51,19 +51,22 @@ nav ul {
 :is(h2, h3):not(:first-child) {
   margin-top: 3rem;
 }
+:is(h4, h5, h6):not(:first-child) {
+  margin-top: 2rem;
+}
 
 h1 {
   font-size: 3.5rem;
   font-weight: bold;
 }
 
 h2 {
-  font-size: 34px;
+  font-size: 32px;
   font-weight: bold;
 }
 
 h3 {
-  font-size: 22px;
+  font-size: 24px;
```

Comment:
Should CSS and TS changes be included in a docs content PR? I feel like they should be split up, no?

Reply:
they definitely should, this is me being lazy :) I'll revert and add in a separate PR